### PR TITLE
#15: Add githubPublishTo setting

### PR DESCRIPTION
**Pull request by xerial**
An implementation for #15

### CODE REVIEW

File: src/main/scala/sbtghpackages/GitHubPackagesPlugin.scala [hunk 1]
```diff
@@ -55,7 +55,8 @@ object GitHubPackagesPlugin extends AutoPlugin {
     })
 
   val packagePublishSettings = Seq(
-    publishTo := {
+    publishTo := githubPublishTo.value,
+    githubPublishTo := {
```
Comment (djspiewak) [inline src/main/scala/sbtghpackages/GitHubPackagesPlugin.scala:59]: 👍

**Comment by xerial** [comment]
Thanks. Applied the suggestion :)

**Comment by djspiewak** [comment]
Merged; thanks!

**Comment by mkurz** [comment]
@djspiewak When will this feature make it into a release?

**Comment by djspiewak** [comment]
I can make a release today if you like. :-) I didn't have anything specific in flight; just hadn't made a new release after the merge.

**Comment by mkurz** [comment]
@djspiewak If you could cut a release today, that would be nice. I have use cases for that feature here as well. Thanks!

**Comment by djspiewak** [comment]
@mkurz Belatedly… released 0.5.1!

**Comment by mkurz** [comment]
Thanks!

**Comment by mkurz** [comment]
This is not working correctly for me. When I set
```
publishTo := githubPublishTo.value
```
in `build.sbt` I get:

```
[error] Cyclic reference involving 
[error]    ProjectRef(uri("file:/home/mkurz/myproject/"), "sbt-build-utils") / githubPublishTo
[error]    ProjectRef(uri("file:/home/mkurz/myproject/"), "sbt-build-utils") / publishTo
[error] Use 'last' for the full log.
```

@xerial Does it work for you?

**Comment by djspiewak** [comment]
I'm seeing this same problem, @mkurz. I'll see if I can diagnose

**Comment by djspiewak** [comment]
@mkurz It's fixed in 0.5.2

**Comment by mkurz** [comment]
@djspiewak Thanks, now it works :+1:

**Comment by xerial** [comment]
@djspiewak @mkurz Thanks for fixing for cyclic reference.

**Pull request merged**